src/python/pants/backend/python/goals/debug_goals.py

```diff
@@ -65,6 +65,7 @@ class DumpPythonSourceAnalysisSubsystem(GoalSubsystem):
 
 class DumpPythonSourceAnalysis(Goal):
     subsystem_cls = DumpPythonSourceAnalysisSubsystem
+    environment_behavior = Goal.EnvironmentBehavior.LOCAL_ONLY  # TODO(#17129) — Migrate this.
```

Comment:
No need to migrate. This is right

```suggestion
    environment_behavior = Goal.EnvironmentBehavior.LOCAL_ONLY
```

Reply:
oh, we should probably change the other debug goals then?